### DISCUSSION TITLE
About iOS Minimum Version Requirement

### DISCUSSION THREAD

**Discussion by lazjedi** [anyproto/contributors#222] [Anytype apps]
Hi, We want to use AnyType on iPhone 7 but the latest iOS version of iPhone 7 is 15.8. AnyType has minimum iOS 16 requirements. Can you somehow downgrade this to iOS 15? Alternative note apps support lower iOS versions.

**Comment by joanna-murzyn** [comment]
Bump up.

Reducing support for older devices contributes to planned hardware obsolesce by software.
This in long term push customers to change their hardware even it still function properly.

**Reply by lazjedi** [reply]
They said on the official forum that they can't do that.

**Reply by joanna-murzyn** [reply]
Is there any particular reason why?

**Reply by lazjedi** [reply]
API versions used by the app are not supported.
